### PR TITLE
64bit nfs xfer

### PR DESCRIPTION
**Pull request by roblatham00**
test:jenkins/ch3/all

### CODE REVIEW

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by roblatham00** [comment]
test:jenkins/ch3/all

**Comment by raffenet** [comment]
@roblatham00 I know we had a lot of Jenkins issues in the past few weeks. Hopefully one more test won't hurt before review.
test:jenkins/ch3/tcp

**Comment by raffenet** [comment]
Sigh. Let's try that again. test:jenkins/ch3/tcp

File: src/mpi/romio/adio/common/lock.c [hunk 1]
```diff
@@ -179,13 +179,16 @@ int ADIOI_Set_lock(FDTYPE fd, int cmd, int type, ADIO_Offset offset, int whence,
 }
 #endif
 
-#if defined(ROMIO_XFS)
 int ADIOI_Set_lock64(FDTYPE fd, int cmd, int type, ADIO_Offset offset,
                      int whence,
 	             ADIO_Offset len) 
 {
     int err, error_code;
+#ifdef _LARGEFILE64_SOURCE
```
Comment (raffenet) [inline src/mpi/romio/adio/common/lock.c]: Is this `#ifdef` necessary? Looking at calls to this function, it seems a 64-bit lock cmd is always passed.

**Comment by roblatham00** [comment]
that's a workaround for strict  solaris and freebsd  -- those platforms
either don't have the 64 bit variants or just do it without a
LARGEFILE_SOURCE define.

**Pull request merged**